### PR TITLE
Airflow can now make you drop your hat

### PR DESCRIPTION
**Pull request by ShiftyRail**
I just thought of that when I was biking outside and the wind made me drop my hat

:cl:
- rscadd: Airflow can now make you drop your hat, sometimes.

### CODE REVIEW

**Comment by soyconsumer** [comment]
Biking with a fedora again, huh?

**Comment by ShiftyRail** [comment]
@soyconsumer A normal cap, for sun, actually.

**Comment by boy2mantwicethefam** [comment]
my opinion to this pr sums up as yesn't

File: code/ZAS/Airflow.dm [hunk 1]
```diff
@@ -143,6 +164,9 @@ atom/movable/GotoAirflowDest(n)
 		return
 	if(ismob(src))
 		to_chat(src, "<span class='warning'>You are sucked away by airflow!</span>")
+		if (ishuman(src))
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/ZAS/Airflow.dm]: Doesn't affect MoMMIs huh?? Damn c*ders and their silicon fetish

File: code/ZAS/Airflow.dm [hunk 2]
```diff
@@ -39,6 +39,19 @@ atom/movable/GotoAirflowDest(n)
 
 */
 
+// Hats which can't be removed by airflow
+var/list/unremovable_hats = list(
```
Comment (DamianX) [inline code/ZAS/Airflow.dm]: this is gay, make it a proc/var on the helmet

**Comment by Dacendeth** [comment]
Does this only happen if you get knocked down?

**Comment by ShiftyRail** [comment]
Yes, and if you get pulled away
Also don't merge this I have things to fix

**Comment by ShiftyRail** [comment]
This was a fun idea but it's not getting anywhere

**Pull request closed without merging**